typescript/sdk/src/token/CosmosNativeWarpRouteReader.ts

```
@@ -0,0 +1,194 @@
+    } as HypTokenConfig;
+  }
+
+  async fetchRemoteRouters(warpRouteAddress: Address): Promise<RemoteRouters> {
+    const { remote_routers } =
+      await this.cosmosProviderOrSigner.query.warp.RemoteRouters({
+        id: warpRouteAddress,
+      });
+
+    const routers = remote_routers.reduce(
+      (value, router) => ({
+        ...value,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)